soltesz-lab-to-nwb/convert_data.py

```diff
@@ -0,0 +1,89 @@
+"""Authors: Cesar Echavarria and Ben Dichter."""
+# load necessary modules
+from pathlib import Path
+import os
+from isodate import duration_isoformat
+from datetime import timedelta, datetime
+from datanwbconverter import DataNWBConverter
+
+from pprint import pprint
+
+# Hard-coded parameters
+sampling_rate = 10000  # sampling rate for ephys data
+
+# # Point to the base folder path for data
+base_path = Path("/Users/cesar/Documents/CatalystNeuro/Farrell_SuM_2021/SuM_2-Photon/")
+
+# #change these for the file of interest
+animal = "JF_141"
+suffix = "_006"
+animal_path = os.path.join(base_path, animal)
+
+# Name the NWBFile and point to the desired save path
+# nwbfile_path = os.path.join(animal_path ,'FullTesting.nwb')
+nwbfile_path = os.path.join(animal_path, "FullTesting.nwb")
+
+# Point to the various files for the conversion
+speed_file_path = os.path.join(animal_path, "GRIN" + animal[2:] + suffix + ".speed.npy")
+opto_file_path = os.path.join(animal_path, 'GRIN' + animal[2:] + suffix+ '.sbx')
```

Comment:
```suggestion
opto_file_path = os.path.join(animal_path, f"GRIN{animal[2:]}{suffix}.sbx")
```
I think this syntax is a little cleaner